packages/frontend/lib/index.ts

```diff
@@ -16,10 +15,10 @@ export default class Nango {
     private hostBaseUrl: string;
     private websocketsBaseUrl: string;
     private status: AuthorizationStatus;
-    private publicKey: string | undefined;
+    private publicKey: string;
     private debug = false;
 
-    constructor(config: { host?: string; websocketsPath?: string; publicKey?: string; debug?: boolean } = {}) {
+    constructor(config: { host: string; websocketsPath?: string; publicKey: string; debug?: boolean }) {
```

Comment:
`host` should stay optional, no?

Reply:
Yup, updated with https://github.com/NangoHQ/nango/pull/760/commits/45371af3fad39237cd6ced1601c3a6d7eb3c1610